tests/integration/test_definitions.py

```diff
@@ -45,6 +45,19 @@ def define_tests():
             "local",
         ),
     }
+    server_tests = {
+        "merlin server init": ("merlin server init", HasRegex(".*successful"), "local"),
+        "merlin server start/stop": (
+            "merlin server start; merlin server status; merlin server stop",
+            [
+                HasRegex("Server started with PID [0-9]*"),
+                HasRegex("Merlin server is running"),
+                HasRegex("Merlin server terminated"),
```

Comment:
Good! Once we have an easy configuration, it'd be good to have a test that confirms that Merlin can connect to the right server, eg that "merlin info" returns ok

Reply:
Agreed, something that might be useful is for a command or an easy interface where users will be able to switch freely between using the merlin server and using an external server like before. The only way that I have been able to do it so far is to change the app.yaml file manually within ~/.merlin/. It might be good to have something like "merlin server set local" and "merlin server set remote" to allow users to switch between the two without having to manually change the file.

Reply:
Let's make an issue on this